config/locales/en.yml

```diff
@@ -162,13 +219,35 @@ en:
         add: Add
         update: Update
         current_admins:
-          actions: Action
-          email: Email address
           header: Current Account Administrators
+          all_users: All Users
+          email: Email address
+          actions: Action
           no_admins: No administrators exist
         edit: Edit Account
         header: Manage Account
         remove: Remove
+    users:
```

Comment:
remove line 230-250 for this and all other languages

Reply:
The 20 line code block starting with users: has been deleted.